index.js

```diff
@@ -0,0 +1,196 @@
+     "technologies":["Ruby on rails","css","javascript","html"],
+     "live":"ldouglasot.github.io",
+     "source":"https://github.com/LDouglasOT/ldouglasot.github.io"
+  },{
+    "name":"Multi-Post Stories</span><span>Gain+Glory",
+     "description":"This is a website developed using html,css and javascript following best cl/ci practices", 
+     "featured":"../Assets/Images/header-illsutration-mobile@2x.png", 
+     "technologies":["Ruby on rails","css","javascript","html"],
+     "live":"ldouglasot.github.io",
+     "source":"https://github.com/LDouglasOT/ldouglasot.github.io"
+  },{
+    "name":"Multi-Post Stories</span><span>Gain+Glory",
+     "description":"This is a website developed using html,css and javascript following best cl/ci practices", 
+     "featured":"../Assets/Images/header-illsutration-mobile@2x.png", 
+     "technologies":["Ruby on rails","css","javascript","html"],
+     "live":"ldouglasot.github.io",
+     "source":"https://github.com/LDouglasOT/ldouglasot.github.io"
+  },{
+    "name":"Multi-Post Stories</span><span>Gain+Glory",
+     "description":"This is a website developed using html,css and javascript following best cl/ci practices", 
+     "featured":"../Assets/Images/header-illsutration-mobile@2x.png", 
+     "technologies":["Ruby on rails","css","javascript","html"],
+     "live":"ldouglasot.github.io",
+     "source":"https://github.com/LDouglasOT/ldouglasot.github.io"
+  },
+
+]
+
+window.addEventListener('load', ()=>{
+
+  const untitlelist=document.querySelector('.works')
+
+  const li=document.createElement('ul')
+  li.classList.add('works-ul')
+  const insertitems=(name,description,featured,live,source,technologies)=>{
+  return `
+
+  <div class="top-notch">
+    <img class="imagedescription" src="https://wallpaperaccess.com/full/220746.jpg" alt='capstone' />
+    <div class="top-notch-content">
+      <h3 class="top-tag">
+        <span>${name}</span>
+      </h3>
+      <ul class="top-notch-content-ul">
+      
+        <li class="top-notch-content-ul-li">${technologies[0]}</li>
+        <li class="top-notch-content-ul-li">${technologies[1]}</li>
+        <li class="top-notch-content-ul-li">${technologies[2]}</li>
+        <li class="top-notch-content-ul-li">${technologies[3]}</li>
+      </ul>
+      <br />
+      <button class="top-notch-content-a" id='seeproject'>See Project</b>
+      <br />
+    </div>
+  </div>
+ 
+  `
+   }
+   projectlist.map((item)=>{
+    const lists=document.createElement('li')
+    lists.classList='list-item'
+    lists.innerHTML=insertitems(item.name, item.description, item.featured, item.live, item.source,item.technologies)
+    li.appendChild(lists)
+   })
+   untitlelist.append(li)
+});
+
+
+
+window.onload = function() {
+if (window.matchMedia("(max-width: 680px)").matches) {
+const btnclick=document.querySelector('#seeproject')
+console.log(btnclick)
+
+let pop = document.createElement('div')
+  pop.classList.add('popout')
+  pop.innerHTML=`
```

Comment:
Kindly check and make sure that the pop-up is responsive across both the mobile and desktop versions to match the design

Your mobile design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-14 11-25-15](https://user-images.githubusercontent.com/65233860/195838154-0798fd71-a450-4a51-8a75-20594b36eef3.png)  |  ![Screenshot from 2022-10-14 11-37-26](https://user-images.githubusercontent.com/65233860/195838227-bab9acf2-8ef3-4758-9007-fafceda2e983.png)